README.md

```diff
@@ -16,7 +16,16 @@ Scripts for updating and debugging Kafka can be found [here](https://github.com/
 | Environment variable      | Default                              | Description
 | ------------------------- | -------------------------------------| ------------------------------
 | BIND_ADDR                 | :22200                               | The port to bind the application healhcheck endpoint to
-| KAFKA_ADDR                | http://localhost:9092                | The address of the kafka Instance
+| KAFKA_LEGACY_ADDR         | `localhost:9092`                     | The addresses of the kafka brokers (CSV) - non-TLS
+| KAFKA_LEGACY_VERSION      | `1.0.2`                              | The version of Kafka - non-TLS
+| KAFKA_ADDR                | `localhost:9092`                     | The addresses of the kafka brokers (CSV)
+| KAFKA_VERSION             | `1.0.2`                              | The version of Kafka
+| KAFKA_SEC_PROTO           | _unset_                              | if set to `TLS`, kafka connections will use TLS [1]
```

Comment:
Maybe link `[1]` to notes bullet point 1 - not necessarily obvious why this is here unless you review PR or look at code changes

Reply:
is this any better?

Reply:
You can use this as a reference (https://github.com/ONSdigital/dp-dimension-search-api/pull/88). This was reviewed by Nathan as well and approved

Reply:
Thanks Justin - yes I was thinking of the same changes made here

Reply:
my markdown linter complains about inline-HTML for your method, so I prefer my method 😛 
(frankly, I think if people don't understand the original footnote syntax, then 🤷 but here we are...)